client/blocks/plugins-scheduled-updates/schedule-form-paths.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+	__experimentalText as Text,
+	__experimentalInputControl as InputControl,
+	Button,
+	Flex,
+	FlexItem,
+} from '@wordpress/components';
+import { check, plus, closeSmall } from '@wordpress/icons';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useState, useCallback } from 'react';
+import { MAX_SELECTABLE_PATHS } from './config';
+import { useSiteSlug } from './hooks/use-site-slug';
+import { validatePath } from './schedule-form.helper';
+
+interface Props {
+	paths?: string[];
+	borderWrapper?: boolean;
+}
+export function ScheduleFormPaths( props: Props ) {
+	const translate = useTranslate();
+	const siteSlug = useSiteSlug();
+	const { paths: initPaths = [], borderWrapper = true } = props;
+
+	const [ paths, setPaths ] = useState( initPaths );
+	const [ newPath, setNewPath ] = useState( '' );
+	const [ newPathError, setNewPathError ] = useState( '' );
+
+	const removePath = useCallback(
+		( index: number ) => {
+			setPaths( paths.filter( ( _, i ) => i !== index ) );
+		},
+		[ paths ]
+	);
+
+	const onNewPathSubmit = useCallback( () => {
+		const pathError = validatePath( newPath );
+		setNewPathError( pathError );
+
+		if ( pathError ) {
+			return;
+		}
+
+		setPaths( [ ...paths, newPath ] );
+		setNewPath( '' );
+	}, [ newPath, paths ] );
+
+	return (
+		<div className="form-field form-field--paths">
+			<label htmlFor="paths">{ translate( 'Test URL paths' ) }</label>
+
+			{ newPathError ? (
+				<Text className="validation-msg">{ newPathError }</Text>
+			) : (
+				<Text className="info-msg">
+					{ translate(
+						'Your schedule will test your front page for errors. Do you want to add additional paths?'
```

Comment:
Can you help clarify the meaning of this string? It's unclear to us how the schedule will test the front page for errors.

Does this mean that the front page will be tested for errors based on the scheduled frequency or after the scheduled updates?

Reply:
The front page will be tested right after the scheduled updates.

That description has been recently updated to "Your schedule will test your front page for errors. Add up to %(maxPaths)s additional paths to test:" in PR: https://github.com/Automattic/wp-calypso/pull/89850